docs/tutorials/05_qaoa.ipynb

```diff
@@ -0,0 +1,392 @@
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[1 1 0 0]\n",
+      "Objective value computed by QAOA is 3\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.primitives import Sampler\n",
+    "from qiskit.quantum_info import Pauli, Statevector\n",
+    "from qiskit.result import QuasiDistribution\n",
+    "\n",
+    "from qiskit_algorithms import QAOA\n",
+    "from qiskit_algorithms.optimizers import COBYLA\n",
+    "\n",
+    "from qiskit_algorithms.utils import algorithm_globals\n",
+    "\n",
+    "sampler = Sampler()\n",
+    "\n",
+    "def sample_most_likely(state_vector: QuasiDistribution | Statevector) -> np.ndarray:\n",
+    "    \"\"\"Compute the most likely binary string from state vector.\n",
+    "    Args:\n",
+    "        state_vector: State vector or quasi-distribution.\n",
+    "\n",
+    "    Returns:\n",
+    "        Binary string as an array of ints.\n",
+    "    \"\"\"\n",
+    "    if isinstance(state_vector, QuasiDistribution):\n",
+    "        values = list(state_vector.values())\n",
+    "    else:\n",
+    "        values = state_vector\n",
+    "    n = int(np.log2(len(values)))\n",
+    "    k = np.argmax(np.abs(values))\n",
+    "    x = bitfield(k, n)\n",
+    "    x.reverse()\n",
+    "    return np.asarray(x)\n",
+    "\n",
+    "algorithm_globals.random_seed = 10598\n",
+    "\n",
+    "optimizer = COBYLA()\n",
+    "qaoa = QAOA(sampler, optimizer, reps=2)\n",
+    "\n",
+    "result = qaoa.compute_minimum_eigenvalue(qubit_op)\n",
+    "\n",
+    "x = sample_most_likely(result.eigenstate)\n",
+    "\n",
+    "print(x)\n",
+    "print(f'Objective value computed by QAOA is {objective_value(x, w)}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The outcome can be seen to match to the value computed above by brute force. But we can also use the classical `NumPyMinimumEigensolver` to do the computation, which may be useful as a reference without doing things by brute force."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[1 1 0 0]\n",
+      "Objective value computed by the NumPyMinimumEigensolver is 3\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_algorithms import NumPyMinimumEigensolver\n",
+    "from qiskit.quantum_info import Operator\n",
+    "\n",
+    "npme = NumPyMinimumEigensolver()\n",
+    "result = npme.compute_minimum_eigenvalue(Operator(qubit_op))\n",
+    "\n",
+    "x = sample_most_likely(result.eigenstate)\n",
+    "\n",
+    "print(x)\n",
+    "print(f'Objective value computed by the NumPyMinimumEigensolver is {objective_value(x, w)}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is also possible to use VQE as is shown below"
```

Comment:
> It is also possible to use VQE as is shown below

Look like this was changed over to SamplingVQE and the text never updated. Maybe we should do that and in the print result to avoid any confusion with "VQE". I also see the result here is different from the ones above - maybe thats an equally valid result though